packages/cozy-client/package.json

```diff
@@ -66,7 +66,9 @@
     "cozy-ui": ">34.0.0",
     "react": "^16.7.0",
     "react-native": "^0.65.0",
-    "react-native-inappbrowser-reborn": "^3.5.1"
+    "react-native-google-safetynet": "0.0.4",
+    "react-native-inappbrowser-reborn": "^3.5.1",
+    "react-native-ios11-devicecheck": "0.0.3"
```

Comment:
should be our fork, right?

Reply:
[Fixed](https://github.com/cozy/cozy-client/compare/1100dc38c7a24690f485b99a53b421a7d3e53b09..876e2ca52277c285a8e2bed5437fa6fa6ee58cc7)